qulacs-src/cppsim/gate_named_pauli.hpp

```diff
@@ -115,6 +145,13 @@ class ClsPauliRotationGate : public QuantumGateBase {
      * @param angle 回転角
      * @param pauli 作用させるパウリ演算子
      */
+    /**
+     * \~english Construct
+     * 
+     * Pauli operator used is released in class
```

Comment:
this one might be a minor point,
In my opinion, "release the used Pauli operator from the class "sounds more natural?
because English uses less passive forms than Japanese.